test/addons/rook-external-cluster/start

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+import time
+import shlex
+import io
+import tempfile
+import subprocess as subp
+import os.path as ospa
+import logging
+
+import drenv
+from drenv import kubectl
+from drenv import commands
+
+
+# Update this when upgrading rook.
+VERSION = "release-1.10"
+BASE_URL = f"https://raw.githubusercontent.com/rook/rook/{VERSION}/deploy/examples"
+ROOK_CEPH_NAME_SPACE = "rook-ceph"
+
+
+def rook_connect_to_external_storage_cluster(cluster_details, managed_cluster):
+    env = os.environ.copy()
+    script_name = ''
+
+    _, script_name = tempfile.mkstemp(prefix='import-external-cluster-', suffix='.sh')
```

Comment:
I think this leaks a file descriptor - not critical since this is a short running script, but usually sign of code copied from somewhere else.